src/Microsoft.SqlTools.ServiceLayer/LanguageServices/PeekDefinition.cs

```diff
@@ -203,9 +225,9 @@ internal string GetSchemaFromDatabaseQualifiedName(string fullObjectName, string
             string[] tokens = fullObjectName.Split('.');
             for (int i = tokens.Length - 1; i > 0; i--)
             {
-                if(tokens[i].Equals(objectName))
+                if (tokens[i].Equals(objectName))
                 {
-                    return tokens[i-1];
+                    return tokens[i - 1];
```

Comment:
does the name always has schema? shouldn't you check for when tokens has only one item in it?